astropy/coordinates/sites.py

```diff
@@ -0,0 +1,138 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Observatories accessible by the `sites` module originate from the IRAF
+Observatory Database, and are stored in ``data/observatories.json``.
+Longitudes are listed with positive to the West.
```

Comment:
This confused the heck out of me, since I couldn't find where the sign is changed (since default is positive to east). But then I saw that the json actually has longitudes between 0 and 360, measured east of greenwich. Is this indeed the case?  If so, does this mean it changed the original IRAF file?  In any case, please correct this comment!